code/modules/admin/verbs/striketeam.dm

```diff
@@ -37,33 +52,35 @@ GLOBAL_VAR_INIT(sent_strike_team, 0)
 			break
 
 	// Find ghosts willing to be DS
-	var/image/source = image('icons/obj/cardboard_cutout.dmi', "cutout_deathsquad")
-	var/list/commando_ghosts = pollCandidatesWithVeto(src, usr, COMMANDOS_POSSIBLE, "Join the DeathSquad?",, 21, 60 SECONDS, TRUE, GLOB.role_playtime_requirements[ROLE_DEATHSQUAD], TRUE, FALSE, source = source)
+	var/list/commando_ghosts = list()
+	if(alert("Would you like to custom pick your Deathsquad?",, "Yes", "No") == "Yes")
+		var/image/source = image('icons/obj/cardboard_cutout.dmi', "cutout_deathsquad")
+		commando_ghosts = pollCandidatesWithVeto(src, usr, commando_number, "Join the DeathSquad?",, 21, 60 SECONDS, TRUE, GLOB.role_playtime_requirements[ROLE_DEATHSQUAD], TRUE, FALSE, source = source)
+	else
+		var/image/source = image('icons/obj/cardboard_cutout.dmi', "cutout_deathsquad")
+		commando_ghosts = shuffle(SSghost_spawns.poll_candidates("Join the Deathsquad?",, GLOB.responseteam_age, 60 SECONDS, TRUE, GLOB.role_playtime_requirements[ROLE_DEATHSQUAD], TRUE, FALSE, source = source))
```

Comment:
don't `shuffle` it here - the order is randomzied already in the following loop (the `var/mob/ghost_mob = pick(commando_ghosts)` line)